### PR TITLE
docs(table): add additional documentation to the table package

### PR DESCRIPTION
**Pull request by jaded-dev**
**Before submitting a pull request,** please make sure the following is done:

1. Fork [the repository](https://github.com/availity/availity-react) and create your branch from `master`.
2. Run `yarn` in the repository root.
3. If you've fixed a bug or added code that should be tested, add tests!
4. Ensure the test suite passes (`yarn test`). Tip: `yarn test --watch TestName` is helpful in development.
5. Make sure your code passed the conventional commits check. Read more about [conventional commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/#summary)

### CODE REVIEW

**Review by jordan-a-young — APPROVED** [review]

**Comment by jaded-dev** [comment]
add additional documentation to the table package

**Comment by LauRoxx** [comment]
I know this was already merged but were you planning on adding an argstable for the remaining table components?

File: packages/table/src/Table.tsx [hunk 1]
```diff
@@ -22,37 +22,75 @@ import { TableSortOption } from './types';
 import TableContent from './TableContent';
 
 type HeaderProps = {
+  /** This makes the column sticky to the table. This works best when inside a scrollable container. Not supported in IE11. */
   sticky: boolean;
 } & React.HTMLAttributes<HTMLElement>;
 
 export type CommonTableProps<T extends IdType> = {
+  /** This is a unique id that is prepended to the table and nested table elements. **/
```
Comment (LauRoxx) [inline packages/table/src/Table.tsx:30]: Not the biggest deal, but documentation comments are /** */, it looks like you're adding an extra asterisk at the end which will end up adding an asterisk to the description text.

File: packages/table/src/Table.tsx [hunk 2]
```diff
@@ -22,37 +22,75 @@ import { TableSortOption } from './types';
...
   selectable?: boolean;
+  /** This function determines if a row in a selectable table can be
+   * selected. By default if no function is provided to this property,
+   * all rows in the table are selectable. **/
   getCanSelectRow?: (record: T) => boolean;
+  /**  **/
   sortable?: boolean;
+  /** This boolean determines whether the table is paged or not. This
+   * works with the usePagination hook that is documented in react-table.
+   * This defaults to false. **/
```
Comment (LauRoxx) [inline packages/table/src/Table.tsx:81]: The args table has a separate column to communicate default values so no need to add bulk to the description.

**Pull request merged**